.github/workflows/docker_img.yaml

```diff
@@ -42,11 +42,6 @@ jobs:
                     - "-esp32"
                     - "-esp32-qemu"
                     - "-infineon"
-                    # NOTE: imx image requires too much space for GitHub-hosted runners. It fails with:
```

Comment:
Why remove this comment?

Reply:
Because I had it working, then it broke again, and didn’t want to re-spin CI… I’ll fix and force in.

Reply:
Fixed